source/Pages/best_practices.rst

```diff
@@ -0,0 +1,160 @@
+----------------------------------------------
+
+When transferring data from/to Spider your experience will vary depending on the client, protocol and paremeters you chooce. Thus, for data transfers we suggest you to use Rclone. Rclone is a command line tool that works on many platforms and it can talk to many storage systems, including dCache. Some advantages of Rclone are that it can sync directories, like rsync does, and it uses parallel transfers, 4 by default, to get a better performance when copying directories. More information about using Rclone, for example with dCache, can be found in our :ref:`ADA <transfer-data-rclone>` section.
+
+
+When you need to tar or zip many small files on Spider, this can be very slow on the local CephFS filesystem and can take several hours. In such cases it may be better to copy the files temporarily in memory as it will speed up these operations remarkably. When the files are copied from CephFS into memory in a parallel way, it will be much faster than tar which does it one by one. Once the files are in the page cache of the node, the tar process is a lot faster. When using this option please keep in mind that memory is limited and shared with other processes and that it is temporary. An example for using the shared memory to tar and process a file can be found in :ref:`Shared memory <shared-memory>`.
+
+For advanced users, who are familiar with mpi operations, we also offer an a MPI-based tool for managing datasets such as copying files across the different home and project space folders on the local file system. The MPI-based tool is much faster and efficient than the common `cp` operations. Example usage for parallel copying of files using this method can be found in the :ref:`mpifileutils <mpi-file-utils>` section.
+
+
+.. _running-many-jobs:
+
+Running a large amount of jobs 
+==============================
+
+When running a large amount of jobs it can be difficult to keep track of the state of these jobs or resume failed tasks that were prematurely canceled due to time limit. Another challenge when designing high-throughput workflows that execute a specific application for many different parameter combinations, is reducing the large scheduling overhead and waiting times in the queue. 
+
+
+An overview of the features and capabilities of some of the options for running a large amount of jobs on Spider is presented below.
+
+==============================================   ==============   ==============   =============
+Feature                                          Array jobs       Picas            Snakemake
+==============================================   ==============   ==============   =============  
+High speed & Low load on the system              No               Yes              Somewhat  
+Scales to hundreds, thousands of jobs and more   No               Yes              Somewhat
+Available outside Spider                         No               Yes              No
```

Comment:
Correct me if I am wrong, isn't Snakemake also available elsewhere?

Reply:
Good point. It is rephrased as 'transcends spider' to explain whether it is available in single slurm cluster or across many.